CONTRIBUTING.md

```diff
@@ -0,0 +1,95 @@
+
+Please don't use the issue tracker for support questions. You can use the [Arbor Discussions](https://github.com/arbor-sim/arbor/discussions)
+page for this.
+
+## Github workflow
+
+Arbor uses Git for version control and Github to host its code repository and nearly
+all of its infrastructure.
+
+* If you're not familiar with Git or Github, please have at look at
+[this introduction](https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github/set-up-git).
+* Make sure you have a [GitHub account](https://github.com/signup/free).
+
+### Start a discussion
+
+You can use the [Arbor Discussions](https://github.com/arbor-sim/arbor/discussions) to help other users of Arbor,
+or get help with you own simulations. Also, we're eager to discover what you're using Arbor for, so don't hesitate
+to share your Arbor simulations or publications!
+
+### Filing an issue
+
+If you have found a bug or problem in Arbor, or want to request a feature, you can use our
+[issue tracker](https://github.com/arbor-sim/arbor/issues). If you issue is not yet filed in the issue tracker,
+please do so and describe the problem, bug or feature as best you can. You can add supporting data, code or documents
+to help make your point.
+
+### Making a pull request
+
+If you want to contribute code that implements a solution or feature, the workflow is as follows:
+
+1. Fork the Arbor repository
+2. Create a branch off of master and implement your feature
+3. Make a pull request (PR) and refer to the issue(s) that the PR addresses.
+4. An administrative matter: Arbor requests [an explicit copyright assignment](https://en.wikipedia.org/wiki/Copyright_transfer_agreement)
+on your contribution. This grants us the right to defend copyright on your contributions on your behalf,
+and allows us to change the license on your contribution to another open source license should that become
+necessary in the future. You can upload a filled out copy of the agreement as a file attached to the PR, or
+if you prefer not to disclose private information, you can send it to <mailto:arbor-sim@fz-juelich.de>.
+    * [Please find the Copyright Transfer Agreement here](https://github.com/arbor-sim/arbor-materials/tree/master/copyright-transfer-agreement).
+5. A PR on Github is automatically tested by our CI bot called Travis. You can also run these tests yourself
+by building them first (`make tests`) and then running them (`./bin/*unit*`).
+6. A member of the Arbor development team will review your contribution. If they approve,
+your PR will be merged! Normally this should happen within a few days.
+
+Refer to the [Github documentation](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request)
+for more explanation of the Git workflow described above.
+
+### Reviewing a PR
+
+Each pull request is reviewed. Some tips to help you get a positive review:
```

Comment:
I actually meant to keep this line the same as before: `Each pull request is reviewed according to these guidelines:`
but to add: `You can use labels to categorize your PR. For very large PRs, it is likely the reviewer will have
many questions and comments. It can be helpful to reach out on our Slack, Discussions or by email
beforehand if you've big things planned for Arbor!` to the list of tips right underneath.

Reply:
Wouldn't that put it back to what it was? I responded to your point about adding `Some tips to help you get a positive review`, or do you mean to make a flat list out of the section?

Reply:
In lines 74-75 you already had `Some tips to help you get a positive review`. I was just suggesting to move lines 71-73 till after line 75 and reword them so they're a tip. And keep line 69 as: `Each pull request is reviewed according to these guidelines:`

Reply:
OK!